lms/templates/main.html

```diff
@@ -101,9 +101,8 @@
     else:
         header_file = microsite.get_template_path('navigation.html')
 
-    google_analytics_file = microsite.get_template_path('google_analytics.html')
```

Comment:
Right, for those who don't want GTM, they will just have that included template as blank, for those who don't want GA, they will just have that template snippet as blank.

But both includes need to be in this file. GA in the <head> region and GTM at the top of the <body>